### ISSUE TITLE
part-pretext

### ISSUE THREAD

**Issue by here101** [sargdsra/part-pretext#1]
Dear techer,
       Your code is perfect,I just want to know what is the part pretext ,Which paper put forward the method. I will better understand the code if I know the answer.

Thanks your reply forward!

**Comment by sargdsra** [comment]
Dear here101,
Thank you. The part pretext task is a task in that task the image will be broken into 4 or 9 patches and every patch have a label from 0 to 3 for 4 patches state or from 0 to 8 for 9 patches state. It is near to the Jigsaw task.

**Comment by here101** [comment]
> 

I know the meaning of part, may I ask if there is a corresponding paper? I still don't understand many things in the code.
Does this code mean that an image is divided into 4 equal parts or 9 equal parts, and then data enhancement operations such as random clipping and flipping of each equal part of the image are carried out, and then the model predicts which equal part the image fragment belongs to?
Thank you for your reply.

**Comment by sargdsra** [comment]
Yes! You said exactly correct. I don't know exactly the corresponding paper for part pretext task but I mention this paper "Self-Supervised Learning of Pretext-Invariant Representations".
